packages/addons/script/steamlink-rpi/package.mk

```
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="steamlink-rpi"
+PKG_VERSION="1.0"
+PKG_REV="100"
+PKG_ARCH="arm"
+PKG_ADDON_PROJECTS="RPi2"
```

shouldn't it 
```suggestion
PKG_ADDON_PROJECTS="RPi2 RPi4"
```

and also remove `PKG_ARCH="arm"` ?

---

I don't have an RPi4 to test if it works, and haven't seen reports of others trying it.